src/OpenTelemetry.Instrumentation.ConfluentKafka/README.md

```diff
@@ -0,0 +1,11 @@
+# Confluent.Kafka client instrumentation for OpenTelemetry
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.ConfluentKafka)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ConfluentKafka)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.ConfluentKafka)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ConfluentKafka)
+[![codecov.io](https://codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib/branch/main/graphs/badge.svg?flag=unittests-Instrumentation.ConfluentKafka)](https://app.codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib?flags[0]=unittests-Instrumentation.ConfluentKafka)
+
+Download the `OpenTelemetry.Instrumentation.ConfluentKafka` package:
+
+```shell
+dotnet add package OpenTelemetry.Instrumentation.ConfluentKafka
```

Comment:
This package, for a while, will be not released as stable. Messaging sem. conv. is still not stable.

```suggestion
dotnet add package OpenTelemetry.Instrumentation.ConfluentKafka  --prerelease
```

Reply:
done